IoTCoreDefaultApp/IoTCoreDefaultApp/Strings/zh-TW/Resources.resw

```diff
@@ -639,7 +639,7 @@
     <comment>Message displayed when Inbound pairing handler registration fails</comment>
   </data>
   <data name="BluetoothInboundRegistrationSucceededFormat" xml:space="preserve">
```

Comment:
change to BluetoothInboundRegistrationFailed is not added.
Change in English:  <value>Failed to register for inbound pairing.</value>